initfiles/componentfiles/configschema/xsd/esp_service_smc.xsd

```diff
@@ -13,6 +13,19 @@
             <xs:attribute name="warnIfFreeMemoryUnder" type="xs:nonNegativeInteger" use="optional" default="5" hpcc:displayName="Warn Free Memory" hpcc:group="Monitoring" hpcc:tooltip="Available memory under this value is flagged as warning in monitoring output"/>
         </xs:attributeGroup>
 
+        <xs:attributeGroup name="elk">
+            <xs:attribute name="integrateKibana" type="xs:boolean" use="required"  hpcc:defaultInCode="true" hpcc:tooltip="Declare if Kibana dashboard to be integrated in ECLWatch"/>
+            <xs:attribute name="kibanaAddress" type="xs:string" use="optional" hpcc:defaultInCode="localhost" hpcc:tooltip="Address (ip/hostname) of Kibana server"/>
+            <xs:attribute name="kibanaPort" type="xs:nonNegativeInteger" use="optional" hpcc:defaultInCode="5601" hpcc:tooltip="Port number of target Kibana server"/>
+            <xs:attribute name="kibanaEntryPointURI" type="xs:string" hpcc:tooltip="Declares the ECLWatch integration entry point URI for the target Kibana dashboard"/>
+            <xs:attribute name="reportLogStashHealth" type="xs:boolean" use="required" hpcc:defaultInCode="true" hpcc:tooltip="Declare if connectivity to LogStash nodes is to be reported in ECLWatch"/>
+            <xs:attribute name="logStashAdresses" type="xs:string" use="optional" hpcc:tooltip="Declares the target LogStash nodes' adresses (semicolon seperated)"/>
+            <xs:attribute name="logStashPort" type="xs:string" use="optional" hpcc:tooltip="Declares the target LogStash nodes' port"/>
```

Comment:
other port declared as xs:nonNegativeInteger

Reply:
I copied what the original PR used. Shall I change it?

Reply:
yes